core/src/it/scala/org/ensime/core/RefactoringHandlerSpec.scala

```diff
@@ -367,23 +325,33 @@ class RefactoringHandlerSpec extends EnsimeSpec
         case default => fail()
       }
 
-      val sdf = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss Z")
-      val t = sdf.format(new Date((new File(file.path)).lastModified()))
-      val diffContents = diffFile.readString()
-      val expectedContents = s"""|--- ${file.path}	${t}
-                                 |+++ ${file.path}	${t}
-                                 |@@ -1,5 +1,5 @@
-                                 |-import scala._
-                                 |-import java.lang.Integer
-                                 |-import scala.Int
-                                 | import java._
-                                 |+import java.lang.Integer
-                                 |+
-                                 |+import scala._
-                                 |  \n""".stripMargin
+      val diffContents = diffFile.readString()(analyzer.charset)
+
+      val relevantExpectedPart = s"""|@@ -1,5 +1,5 @@
+                                     |-import scala._
+                                     |-import java.lang.Integer
+                                     |-import scala.Int
+                                     | import java._
+                                     |+import java.lang.Integer
+                                     |+
+                                     |+import scala._
+                                     |  \n""".stripMargin
+      val expectedContents = expectedDiffContent(file.path, relevantExpectedPart)
 
       diffContents should ===(expectedContents)
       diffFile.delete()
     }
   }
 }
+
+trait RefactoringHandlerTestUtils extends Assertions {
```

Comment:
because I wanted to do bigger refactoring, backed off from it and I've forgotten to remove it, however now I added another helper method which extracts diff from response and fails a test when response is different type than expected, so now the Assertions trait is needed.